src/main/scala/com/fulcrumgenomics/umi/CallMolecularConsensusReads.scala

```diff
@@ -108,6 +108,7 @@ class CallMolecularConsensusReads
    """)
  val minConsensusBaseQuality: PhredScore = 2.toByte,
  @arg(flag="M", doc="The minimum number of reads to produce a consensus base.") val minReads: Int,
+ @arg(doc="The maximum number of reads to use when building a consensus.") val maxReads: Option[Int] = None,
```

Comment:
One thought I have is instead of downsampling what if you just want only families with no and no less than the given # of reads?  I just wonder if you downsample families with `20+` to `n=3` if you aren't biasing the error rate at the end of the day to those that amplified well, which could have their own properties (i.e. amplified well because they have few errors).

Regardless, I think that the usage needs to state that the reads are downsampled if more than this # are found.